tests/test_mathematical.py

```diff
@@ -596,120 +596,131 @@ def test_gradient_y1_dx(self, array, dx):
         assert_array_equal(expected, result)
 
 
+if has_support_aspect64():
+    dtype_list = [numpy.float32, numpy.int64, numpy.int32]
+    ids_list = ["numpy.float32", "numpy.int64", "numpy.int32"]
+else:
+    dtype_list = [numpy.int64, numpy.int32]
+    ids_list = ["numpy.int64", "numpy.int32"]
+
+
 class TestCeil:
     def test_ceil(self):
```

Comment:
I think we can use helper function `get_float_dtypes` here:
```suggestion
    @pytest.mark.parametrize("dtype", get_float_dtypes())
    def test_ceil(self, dtype):
```